subprojects/core/src/main/java/org/gradle/api/internal/file/delete/Deleter.java

```diff
@@ -36,6 +42,7 @@
 
     private static final int DELETE_RETRY_SLEEP_MILLIS = 10;
 
+    private static final int MAX_REPORTED_PATHS = 32;
```

Comment:
This kind of seems like a lot - I'm thinking about long paths that cause the terminal to wrap, etc.  I would think in practice it will mostly be small numbers of files anyways.  I'm not sure what a better limit would be - maybe 16 would be enough?